config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -814,7 +814,7 @@ presubmits:
             - --node-tests=true
             - --provider=gce
             - --runtime-config=api/all=true
-            - --test_args=--nodes=1 --skip="" --focus="\[Feature:InPlacePodVerticalScaling\]"
+            - --test_args=--nodes=8 --skip="" --focus="\[Feature:InPlacePodVerticalScaling\]"
```

This feature label was deleted when this feature was promoted to beta (#128682).
```suggestion
            - --test_args=--nodes=8 --skip="" --focus="Pod InPlace Resize Container"
```

---

Reverting this suggestion after https://github.com/kubernetes/kubernetes/pull/128875